### PR TITLE
Update links in the SECURITY file.

### PR DESCRIPTION
**Pull request by SamirTalwar-DA**
We now:

- link to the release notes as the canonical definition of the "latest release",
- no longer point people to Stack Overflow for general questions, instead favoring the forum, and
- provide a link to GitHub Issues for filing a new issue without security impact.

### Pull Request Checklist

- [x] Read and understand the [contribution guidelines](https://github.com/digital-asset/daml/blob/master/CONTRIBUTING.md)
- [x] Include appropriate tests
- [x] Set a descriptive title and thorough description
- [ ] Add a reference to the [issue this PR will solve](https://github.com/digital-asset/daml/issues), if appropriate
- [x] Include changelog additions in one or more commit message bodies between the `CHANGELOG_BEGIN` and `CHANGELOG_END` tags
- [ ] Normal production system change, include purpose of change in description

NOTE: CI is not automatically run on non-members pull-requests for security
reasons. The reviewer will have to comment with `/AzurePipelines run` to
trigger the build.

### CODE REVIEW

File: SECURITY.md [hunk 1]
```diff
@@ -2,10 +2,17 @@
 
 ## Supported Versions
 
-The latest released version of DAML as per https://github.com/digital-asset/daml/releases is currently being supported with security updates.
+The latest released version of DAML as per https://github.com/digital-asset/daml/releases is
+currently being supported with security updates.
```
Comment (garyverhaegen-da) [inline SECURITY.md:6]: Should this point to the LATEST file instead? The releases page has many snapshot releases we do not plan to patch in any way, even for security issues.

Comment (SamirTalwar-DA) [reply SECURITY.md:6]: Oh, good point. There are also snapshots in LATEST though; perhaps we should be clear that snapshots are not supported?

Comment (SamirTalwar-DA) [reply SECURITY.md:6]: Or we could link to https://github.com/digital-asset/daml/releases/latest.

Comment (cocreature) [reply SECURITY.md:6]: I would suggest to either link to latest as you proposed or to https://docs.daml.com/support/release-notes.html.

Comment (SamirTalwar-DA) [reply SECURITY.md:6]: I'll go with the docs; seems more "official".

**Pull request merged**